### ISSUE TITLE
Update C# "Getting started" text on docs

### ISSUE THREAD

**Issue by JRiad** [strusoft/femdesign-api#960]
Could we just add the paragraph with the information below from Github to the docs page? It is not evident if you start reading in docs how you should run the examples, but the info on Github is helpful

👩‍💻 C#
To properly run the C# examples, we recommend that you clone the entirety of the FEM-Design API repository with Git. You can find instructions for this [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository). You can also download the entire repository as a .zip file and unpack it on your computer.

There are two ways to run the C# examples: either find a specific example in the femdesign-api/FemDesign.Examples/C# folder, then open the project file (.csproj) found there and run the program. Or, open the main solution (femdesign-api.sln) found in the femdesign-api folder, and select which of the example projects you want to run.

**Comment by Marco-Pellegrino** [comment]
Hi @JRiad !

Have you got any question about it?

**Comment by JRiad** [comment]
I think it is not obvious when you are reading the "docs" that the best way to run the examples is to go to github and download. That's what I meant by this comment

**Comment by Marco-Pellegrino** [comment]
I'll have a look at it.

The user does not need to clone the repository to run the examples.

They just need to copy and paste the code in visual studio code as we are showing in "getting started"

**Comment by JRiad** [comment]
That's true. To me it makes more sense to try it out like in the "getting started" page first, but then at some point you want to download all of them and try running different ones. But this is no big issue, I just thought the github way is smoother once you have gotten started
